remote_deployment.py

```
@@ -28,27 +31,46 @@ def run_remote_deployment(data):
-
-        if build == "stage" or build == "candidate" or build == "ga":
-            pull_stage_ga_images(version, build, client=client)
-            full_zip_name = pull_stage_ga_dependency_file(version, build, host_os, host_platform)
+        if version_tuple < (8, 1, 0):
+            if build == "stage" or build == "candidate" or build == "ga":
+                pull_stage_ga_images(version, build, client=client)
+                full_zip_name = pull_stage_ga_dependency_file(version, build, host_os, host_platform)
+            else:
+                if not image_output_file:
+                    logging.info(f"Generating images list for {version}-{build}")
+                    image_list, stdout_err = generate_images_list(version, build)
```

Ruff (0.15.1)

[warning] Unpacked variable `stdout_err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)